etc/picongpu/cori-nersc/a100.tpl

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+# Copyright 2021 Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# PIConGPU batch script for Cori's SLURM batch system
+
+#SBATCH -p !TBG_queue
+#SBATCH -q shared
+#SBATCH --gpus=!TBG_tasks
+#SBATCH --constraint="dgx"
+#S BATCH --reservation="..."
```

Comment:
Hackathon: add reservation
```suggestion
#SBATCH --reservation="gpu_hackathon_dgx"
```
(not to be committed to docs, because specific to our event)